doc/sphinx/replaces.rst

```diff
@@ -36,6 +36,7 @@
 .. |ident_n,k_n| replace:: `ident`\ :math:`_{n,k_n}`
 .. |ident_n| replace:: `ident`\ :math:`_{n}`
 .. |L_tac| replace:: `L`:sub:`tac`
+.. |Ltac| replace:: `L`:sub:`tac`
```

Comment:
The macro above should eventually be removed in favor of this one.

Reply:
Yes, that's the plan.